### ISSUE TITLE
Invalid characters in XML: &#55357;

### ISSUE THREAD

**Issue by fvdpol** [fvdpol/MM-ExportITunesLibraryXml#5]
example xml fragment:

```
<key>1388</key>
<dict>
   <key>Track ID</key><integer>1388</integer>
   <key>Name</key><string>Goblin &#55357;&#56846;</string>
   <key>Artist</key><string>G Jones</string>
   <key>Album</key><string>internet</string>
   <key>Kind</key><string>MPEG audio file</string>
   <key>Size</key><integer>6665570</integer>
   <key>Total Time</key><integer>123062</integer>
   <key>Year</key><integer>2015</integer>
   <key>Date Modified</key><date>2018-07-24T17:24:17</date>
   <key>Date Added</key><date>2018-08-03T14:32:46</date>
   <key>Bit Rate</key><integer>320</integer>
   <key>Sample Rate</key><integer>44100</integer>
   <key>Track Type</key><string>File</string>
   <key>File Folder Count</key><integer>-1</integer>
   <key>Library Folder Count</key><integer>-1</integer>
   <key>Comments</key><string>&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;&#55357;&#56846;</string>
   <key>BPM</key><string>80</string>
   <key>Location</key><string>file://localhost/C:/Users/Alex/Music/usbs/jungle%20trap/G%20JONES%20-%20GOBLIN_SC_Master.mp3</string>
</dict>

```

**Comment by fvdpol** [comment]
The character reference &#55357; denotes a surrogate code point (U+D83D)
xml codes &#55357;&#56846; --> 😎

see: https://unicodebook.readthedocs.io/unicode_encodings.html#utf-16-surrogate-pairs

"Non-BMP characters (range U+10000—U+10FFFF) are stored as “surrogate pairs”, two 16 bits units: an high surrogate (in range U+D800—U+DBFF) followed by a low surrogate (in range U+DC00—U+DFFF)."


Similar issue found in other file with:
```
 <key>Artist</key><string>G Jones</string>
 <key>Album</key><string>&#55357;&#56397;&#55357;&#56397;&#55357;&#56397;</string>
```
xml codes &#55357;&#56397; -->👍

**Comment by fvdpol** [comment]
test in iTunes MacOS:
![screen shot 2018-08-19 at 01 41 06](https://user-images.githubusercontent.com/1499454/44304025-036db500-a351-11e8-864a-3ff25dce8618.png)

written in iTunes XML as:
![screen shot 2018-08-19 at 01 42 05](https://user-images.githubusercontent.com/1499454/44304033-2009ed00-a351-11e8-81c5-831ccb00394b.png)

**Comment by fvdpol** [comment]
should be corrected by commit fcf77817a12e1468f682bced661933b52f3d79ee ... currently in testing

**Comment by fvdpol** [comment]
Confirmed ok
